### DISCUSSION TITLE
Welcome to PlayCanvas Engine Discussions!

### DISCUSSION THREAD

**Discussion by willeastcott** [playcanvas/engine#2600] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kungfooman** [comment]
Game programming is a field that somehow never gets boring, since there is always something new to learn. I loved to extend existing games in the past (using tools like Radiant from id Software + custom scripting). But at some point I always hit the boundaries/limitations of those closed source engines, so I switched over to open source game engines (mostly related to idtech3 like ioquake3)... the only limitation being my time/ability. Still limiting maybe 😅 

Since I like web and game programming I can combine everything by using PlayCanvas. I used to code mostly in C/C++, but JavaScript can often do just the same (and if not yet: wasm) while being easier, highly interactive (devtools) with higher accessibility/reach. There are quite some websites that could profit from a well developed and tested game engine, for all sorts of non-gaming-related issues (education/visualisation, product presentation, local city guides for tourism and other specific issues).

I am looking to learn/grow with the community and hopefully contribute in useful ways (while working on something together can feel more meaningful aswell).

**Comment by keshawkk** [comment]
Hi! I have started using playcanvas recently and was wondering why not make editor free too ?
I am not using it currently but would like to.
Thanks for the reply

**Reply by willeastcott** [reply]
Hi @keshawkk - the Editor *is* free. 😄  You only need to pay if you require private projects.

**Reply by keshawkk** [reply]
Great! thanks for the info

**Comment by keshawkk** [comment]
Can I export Playcanvas games with SCORM to integrate in any LMS.
If there are any examples regarding that please share.

**Reply by willeastcott** [reply]
I don't know what SCORM is. Can you be more specific about what your requirements are please?

**Reply by keshawkk** [reply]
I want to integrate my Playcanvas game with an LMS. Like in unity we have unity3D SCORM for this integration purpose. I was thinking if we can do the same with Playcanvas.

**Reply by willeastcott** [reply]
I would suggest giving it a try and if you run into problems and need changes to PlayCanvas itself, let us know and we'll do our best to help. 😄
